takwimu/context_processors.py

```diff
@@ -152,3 +155,36 @@ def sdgs(request):
     return {
         'sdgs': sdgs,
     }
+
+
+def asset_manifest(request):
+    manifest_filepath = os.path.join(settings.BASE_DIR, 'takwimu/takwimu_ui/build/asset-manifest.json')
+    asset_manifest = {}
+
+    if os.environ.get('DEBUG', 'True') == 'True':
```

Comment:
1. Use `settings` (`from django.conf import settings`) since `settings.py` already uses `os.environ` to load them, and
2. `DEBUG` may be turned on in dev/staging environments and hence it's a bad setting to check for enabling local development dependencies.